src/utils/encryption.ts

```diff
@@ -0,0 +1,39 @@
+import AES from 'crypto-js/aes'
+import encHex from 'crypto-js/enc-hex'
+import { utils } from 'ethers'
+
+function wordsToUint8Array(words: number[]) {
+  const length = words.length
+  const unit8Array = new Uint8Array(length << 2)
+  let offset = 0
+  let word
+
+  for (let i = 0; i < length; i++) {
+    word = words[i]
+    unit8Array[offset++] = word >> 24
+    unit8Array[offset++] = (word >> 16) & 0xff
+    unit8Array[offset++] = (word >> 8) & 0xff
+    unit8Array[offset++] = word & 0xff
+  }
+
+  return unit8Array
+}
+
+function removePaddingBytesFromSeed(seed: Uint8Array): Uint8Array {
+  const length = seed.length
+  const unit8Array = new Uint8Array(length - 16)
+
+  for (let i = 1; i < length - 15; i++) {
+    unit8Array[i - 1] = seed[i]
+  }
```

Comment:
When seed is decrypted, it has 80 bytes instead of 64. And the first byte is for some reason always 0, and the last 15 are just some padding. So removing those bytes produce the original seed.